src/main/java/org/sagebionetworks/web/client/widget/entity/VersionHistoryWidgetViewImpl.java

```diff
@@ -94,6 +98,13 @@ public void setEntityBundle(Entity entity, boolean isShowingOlderVersion) {
 		md5TableHeader.setVisible(!isTable);
 		currentVersionLink.setTargetHistoryToken(DisplayUtils.getSynapseHistoryTokenNoHash(entity.getId()));
 		currentVersionLink.setVisible(isShowingOlderVersion);
+		String currentVersionLinkText;

```

Comment:
May want a shared currentVersionLinkText(Entity entity) method instead of the copy